src/redux/cattegories/categories.js

```diff
@@ -0,0 +1,15 @@
+// Reducer
```

Comment:
- [ ] Kindly make sure to define `STATUS_CHECK` as action types, the value of each action type needs to follow the `ducks pattern`, so it will be something like: `const STATUS_CHECK = bookstore/books/STATUS_CHECK`